Source/Fields/LMCCylindricalCavity.cc

```diff
@@ -322,22 +324,36 @@ namespace locust
 
     	//Assumes probe couples to E_theta of mode. If mode is polarized, transforms angle to reference frame of electron
     	std::vector<double> tEFieldAtProbe;
-    	tEFieldAtProbe.push_back(GetNormalizedModeField(l,m,n,tProbeLocation[0],0).back());
-    	tEFieldAtProbe.push_back(GetNormalizedModeField(l,m,n,tProbeLocation[1],0).back());
+    	tEFieldAtProbe.push_back(GetNormalizedModeField(l,m,n,tProbeLocation[0],0,teMode).back());
+    	tEFieldAtProbe.push_back(GetNormalizedModeField(l,m,n,tProbeLocation[1],0,teMode).back());
```

Comment:
Are we missing some of the TM111 mode here?  It looks like we are only allowing the probe to read out the theta component.  That should work for the TE011 mode, but does it give us the total TM111 mode power?    I am actually wondering if we will need to rethink the probe.  Thoughts?

Reply:
I guess the question is do we want to be able to use Locust to get the total mode power at a given location or do we want to model the response of an actual readout probe. A probe may only be able to couple to some field component (e.g. a wire loop readout would only measure the H field normal to the loop).

Reply:
I've updated both the GetFieldAtProbe() function and the modeAmplitude variable to use the norm of the entire electric field. This remains identical for the TE011 mode (which only has an Etheta component anyway), and while the TM111 mode changes in amplitude its azimuthal polarization remains qualitatively the same as before with respect to the readout probe.